staging/src/k8s.io/kube-scheduler/config/v1beta1/types.go

```diff
@@ -266,44 +275,49 @@ func (c *PluginConfig) encodeNestedObjects(e runtime.Encoder) error {
 // Extender holds the parameters used to communicate with the extender. If a verb is unspecified/empty,
 // it is assumed that the extender chose not to provide that extension.
 type Extender struct {
-	// URLPrefix at which the extender is available
+	// The URL prefix at which the extender is available.
 	URLPrefix string `json:"urlPrefix"`
-	// Verb for the filter call, empty if not supported. This verb is appended to the URLPrefix when issuing the filter call to extender.
+	// The verb for the `filter` call, empty if not supported. This verb is appended to the
+	// urlPrefix when issuing the `filter` call to the Extender.
```

Comment:
```suggestion
	// URL prefix when issuing the `filter` call to the Extender.
```

and in other places?

Reply:
Fixed. Should be using `urlPrefix` to avoid confusion.